buildSrc/src/main/kotlin/org/cqfn/save/buildutils/DockerStackConfiguration.kt

```diff
@@ -49,9 +50,19 @@ fun Project.createStackDeployTask(profile: String) {
 //        }
     }
 
+    tasks.register<Exec>("startMysqlDb") {
+        dependsOn("generateComposeFile")
```

Comment:
What about liquibase update task?

Reply:
Thanks, added. Also had to add two different datasource urls in backend: one to use inside the container and one for liquibase task. The latter should also be used when running backend witout docker too, though.